src/app.js

```diff
@@ -1 +1,130 @@
+import './scss/app.scss';
 
+const input = document.getElementById('location');
+const btn = document.getElementById('submit-loc');
+const btnCelsius = document.getElementById('celsius');
+const btnFahren = document.getElementById('fahrenheit');
+const loader = document.getElementById('loader-container');
+
+const weatherIcon = document.getElementById('weather-icon');
+const infoContainer = document.getElementById('info-container');
+
+const locationInfo = document.querySelectorAll('.location > p');
+const tempInfo = document.querySelectorAll('.temp > div > p');
+const weatherInfo = document.querySelectorAll('.weather> div > p');
+const windSpeedInfo = document.querySelectorAll('.wind-speed > div > p');
+
+const errorDiv = document.createElement('div');
+
+let dataHub = null;
+
+async function fetchWeatherApi (location) {
+  errorDiv.innerHTML = '';
+  try{
+    const api = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${location}&APPID=3200d53ac65b442eb5f439f5613ee06c`);
+    const resp = await api.json();
+    console.log(resp)
+    return resp;
+  }catch (err) {
+    console.log('error: unable to fetch weather info');
+    loader.style.display = 'none';
+    errorDiv.innerHTML = `<h2>Unable to find weather info of ${input.value}</h2>`;
+    infoContainer.appendChild(errorDiv);
+  }
+}
+
+async function getWeatherInfo() {
+  const info = await fetchWeatherApi(input.value);
+  console.log(info);
+  try{
+    dataHub = getJSON(info);
+    displayWeatherInfo(dataHub);
+    loader.style.display = 'none';
+  }catch (err) {
+    loader.style.display = 'none';
+    console.log('Unable to find weather info');
+    loader.style.display = 'none';
+    errorDiv.innerHTML = `<h2>Unable to find weather info of ${input.value}</h2>`;
+    infoContainer.appendChild(errorDiv);
+  }
+  //getJSONOld(info)
+}
+
+const getJSON = (jsonData) => {
+
+  if (jsonData.cod == '404') {
+    return;
+  }
+
+  const location = () => {
+    return {city: jsonData.name, country: jsonData.sys.country};
```

Comment:
As mentioned in the section-specific requirements of Js Curriculum, we ought to use object destructuring to get the values from an object. While in this situation it does not help as much it is good coding practice.
Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/destructuring_requirement.png) from Pathwright that shows this as a requirement.

So you can do the destructuring on line 54 and use variables instead of calling the object. This would be applicable to every where you use the JsonData object